token/program-2022/src/instruction.rs

```diff
@@ -409,6 +431,148 @@ pub enum TokenInstruction {
         /// The freeze authority/multisignature of the mint.
         freeze_authority: COption<Pubkey>,
     },
+    /// Gets the required size of an account for the given mint as a little-endian
+    /// `u64`.
```

Comment:
It makes for a clearer experience from all platforms, since we always give `space` as a `u64`, ie https://github.com/solana-labs/solana/blob/293629bdb581dd101fb286b73db5615dc3993911/sdk/program/src/system_instruction.rs#L313 -- and it seems fine to use up a `u64` if it's just return data